build.sbt

```diff
@@ -119,3 +119,20 @@ lazy val example = project.module
       "white-color"     -> "#FFFFFF"
     )
   )
+
+lazy val `scalaz-test` = project
```

Comment:
We are not interested in a ScalaJS version or is it not possible?

Reply:
I think it can be done later; it's not necessarily trivial